pom.xml

```diff
@@ -6,11 +6,51 @@
     <version>1.0.1-SNAPSHOT</version>
     <name>Solace JMS 1.1 Examples</name>
     <description>Solace Messaging Examples for JMS 1.1</description>
+
+    <properties>
+        <opentelemetry.version>1.19.0</opentelemetry.version>
+        <opentelemetry-alpha.version>1.19.0-alpha</opentelemetry-alpha.version>
```

Comment:
what's the story with the alpha version? Is there another JIRA open to move to a non-alpha version in the future?

Reply:
OpenTelemetry releases `opentelemetry-semconv` and few other as alpha only, as they may be subject to change.